psqlparse/nodes/utils.py

```diff
@@ -11,14 +11,15 @@ def get_node_class(class_name):
     return getattr(module, class_name, None)
 
 
-def build_from_obj(obj):
+def build_from_obj(obj, override_class_name=None):
```

Comment:
We shouldn't be needing to override the class name if we only have the classes from postgres